lib/checks/shared/heading-whitespace-only.js

```diff
@@ -0,0 +1,19 @@
+if (axe.commons.text.accessibleTextVirtual(virtualNode)) {
+	return true;
+}
+
+const spaceSeparatorCharCodes = [
```

Comment:
may be worth using something like this - https://www.npmjs.com/package/is-whitespace-character?

Reply:
I had asked Wilco, and he wanted to just start with the 4 failing cases listed in https://github.com/dequelabs/axe-core/issues/1419